src/System.Net.NetworkInformation/tests/FunctionalTests/System.Net.NetworkInformation.Functional.Tests.csproj

```diff
@@ -32,20 +30,10 @@
       <Link>Common\System.Net\Capability.Sockets.cs</Link>
     </Compile>
   </ItemGroup>
-
   <!-- TODO: Remove after the packages are not using System.Private.Networking. -->
   <ItemGroup>
-    <ProjectReference Include="..\..\..\System.Net.NameResolution\src\System.Net.NameResolution.csproj">
```

Comment:
We're not done with System.PrivateNetworking: the TODO above still applies for Windows.